addons/base_import/static/src/xml/base_import.xml

```diff
@@ -315,4 +314,33 @@ lines composed only of empty cells">
             </div>
         </div>
     </t>
+
+    <div t-name="base_import.progressDialog" class="o_import_progress_dialog text-white">
+        <span class="fa fa-spin fa-circle-o-notch fa-2x mb-2"/>
+        <div t-if="isBatch">
+            <div class="o_import_progress_dialog_batch">
+                <span t-esc="task"/> batch <span class="o_import_progress_dialog_batch_count">1</span> out of <span t-esc="totalBatch"/>...
+                <div class="o_import_progress_dialog_time_left d-none">
+                    <span>Estimated time left:</span>
+                    <span class="o_import_progress_dialog_time_left_text"/>
```

Comment:
No variable / value at init, just to have something ?

Reply:
init value not needed as not displayed

Reply:
Not sure what to put apart from "0" maybe?
As Davideuh said, the parent element is "d-none".